deploy/cloudwash_job.yaml

```diff
@@ -31,6 +31,17 @@ objects:
                   - "-d"
                   - "azure"
                   - "--all"
+                volumeMounts:
+                  - name: config-volume
+                    mountPath: /opt/app-root/src/cloudwash/settings.yaml
+                    subPath: settings.yaml
+                volumes:
+                  - name: config-volume
+                    configMap:
+                      name: cloudwash-config
+                envFrom:
```

Comment:
This block shouldn't be necessary, the expected configmap is getting subPath mounted

Reply:
I removed the config map block only 
did you mean the whole volume block?

Reply:
just the `envFrom` map - but the `volumes` map is out of place too, its not part of Container spec, only `volumeMounts`.   `volumes` is part of the Pod spec, that map starts at L20.